src/output-json-stats.c

```diff
@@ -79,6 +185,20 @@ static json_t *OutputStats2Json(json_t *js, const char *key)
     json_t *value = json_object_iter_value(iter);
     if (value == NULL) {
         value = json_object();
+        /* If multi-tenancy is disabled, it means we have only one engine.
+           In this case, we add engine stats in detect obj.
+           The output will be:
+           "detect":{"engine":{"last_reload":"...","rules_loaded":...,"rules_failed":...}, ...}
```

Comment:
Instead of 2 formats like this:
``` json
"detect":{"engine":{"last_reload":"...","rules_loaded":...,"rules_failed":...}, ...}

"tenancy":[{"id":1,"last_reload":"...","rules_loaded":...,"rules_failed":...},
           {"id":2,"last_reload":"...","rules_loaded":...,"rules_failed":...}]
```

why not a single one like this?
``` json
// single
"detect":[{"id":1,"last_reload":"...","rules_loaded":...,"rules_failed":...}]

// multi
"detect":[{"id":1,"last_reload":"...","rules_loaded":...,"rules_failed":...},
          {"id":2,"last_reload":"...","rules_loaded":...,"rules_failed":...}]
```

Reply:
there is also `alert` field in `detect`, this is the reason why I thought to use
```
"detect":{"engine":[...], "alert":...}
```

Reply:
I'm fine with making it 'detect.engines'. The main point is to have a single logic for both with and without tenants